handbook/communication/code_of_conduct.md

```diff
@@ -0,0 +1,188 @@
+# Code of Conduct
+
+[TOC]
+
+
+## Our pledge
+
+We pledge to make participation in our company, our projects, and our community an open, welcoming, and harassment-free experience for everyone, regardless of ability, age, body size, skin color, culture, education, ethnicity, gender identity and expression, immigration status, level of experience, mental and physical ability, nationality, personal appearance, political beliefs, race, relationship or family status, religion, sex, sexual identity and orientation, or socio-economic status.
+
+We care about the safety and comfort of everyone in our community, including colleagues, contributors, and users.
+
+
+## Team member philosophy
+
+If you work at Sourcegraph or participate in our projects and community, you are important to us. If for any reason you don't feel that,
+
+<p id="gdcalert1" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: undefined internal link (link text: "please let us know"). Did you generate a TOC? </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert2">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+[please let us know](#heading=h.8o2hfb3bhng).
+
+Your ultimate success with Sourcegraph will be a function of your personal agency and the environment we create as a team. We want that environment to set you up for success and enable you to thrive while realizing your professional goals and dreams.
+
+Our mission is to enable everyone to unlock the value of code. For us, that means first helping those who code for a living understand the world of code in which they work. Ultimately, it means making the universe of code accessible to billions of people, most of whom currently cannot partake in the code economy. We believe the ability to make a living from code should be available to all.
+
+We live these values within our own organization: We welcome people from a diverse set of backgrounds, and we want every teammate to grow and flourish at Sourcegraph. We believe that a team with a diverse set of viewpoints and experiences united by our common cause will give us the best chance of realizing our mission.
+
+
+## **Our standards**
+
+We encourage you to:
+
+*   **Be welcoming**. Our communities and teams welcome and support people of all backgrounds and identities.
```

Comment:
I know it's more work, but I'd love to see clean markdown - perhaps we need a Sourcegraph markdown styleguide :) ? I think the preference is to us - instead of * and make sure there is only 1 space between - and the list item.

Reply:
Okie doke, fixed